README.md

```diff
@@ -175,31 +175,34 @@ sql_config:
     query:
 ```
 
-### PostgreSQL
+### PostgreSQL - tested on Linux and windows, up to Python v3.7
 Pre-requisites :
 
-1. Make sure Zegami CLI latest is installed 
+1. Standard requirements - code editor, pip package manager, python 3.7.
+
+2. Make sure Zegami CLI latest is installed 
 ```
 pip install zegami-cli[sql] --upgrade --no-cache-dir
 ```
 _Note: --no-cache-dir avoids some errors upon install_
 
-The login command promtps for username and password which is then used to retrieve a long-lived API token which can be used for subsequent requests. The token is stored in a file in the currenet users data directory.
-Once retrieved all subsequest commands will use the stored token, unless it is specifically overridden wiht the `--token` option
+Test the install with the login command, which promtps for username and password. This is then used to retrieve a long-lived API token which can be used for subsequent requests. The token is stored in a file in the currenet users data directory.
+Once retrieved all subsequent commands will use the stored token, unless it is specifically overridden with the `--token` option
 ```
 zeg login
 ```
 
-2. Install pre-requirements for PostgreSQL connection
+3. Install pre-requirements for PostgreSQL connection
 
 Psycopg2 - https://pypi.org/project/psycopg2/ , http://initd.org/psycopg/
 ```
 pip install python-psycopg2
 ```
 
+_libpq-dev was required for linux, not windows_
 libpq-dev - https://pypi.org/project/libpq-dev/ , https://github.com/ncbi/python-libpq-dev
 ```
-pip install libpq-dev
+sudo apt-get install libpq-dev
```

Comment:
Was this supposed to change back to `sudo apt-get'?

Reply:
Yes this time it was - i remembered I didn't need that for windows

Reply:
Ah OK, I see, it's only a unix requirement